### DISCUSSION TITLE
M1 14" Macbook Pro won't turn on after complete discharge in calibration mode

### DISCUSSION THREAD

**Discussion by alessioperetti** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#948] [Q&A]
Hello guys,
I've been using AlDente Pro for almost 1 year now, currently updated to latest version. OS version is beta 13.4.
As title says, I realized that recently my 14" M1 MBP was acting weird with battery % (sometimes draining very quickly, a couple of times turning off at 40-something%). For this reason, I performed a calibration of the battery using the provided calibration mode within AlDente app.
While plugged in, the battery level fell below 15% without beginning to recharge to 100%; however, I thought that this could be due to the miscalibration of the battery itself. When it dropped to 1%, the laptop finally turned off and now it won't turn on again. The laptop is plugged in, but the MagSafe led is not turning on.
What to do now?
Thanks
Alessio

**Comment by masterzone3d** [comment]
Dear Alessio, could you take a look to the real percentage of the battery by going to preferences panel > battery and click into the (I) icon where percentage will be...
It's very strange that AlDente didn't start to charge from 15% to 100%...maybe some differences from real percentage and OSX percentage. What kind of settings are using to show the main bar perchance?

**Comment by alessioperetti** [comment]
Hello,
Unfortunately it was not really possible for me to troubleshoot the problem
because the laptop was basically dead.
As a follow up to my post, I tried to leave the magsafe plugged for a whole
night to no avail. Since I hadn't another power supply available, I then
tried to recharge the Macbook by connecting it with an USBC to USBA cable
to another desktop computer. I left the thing plugged for almost 36 hours.
Only then, I managed to turn on the MacBook by plugging in the magsafe: the
battery was reading 1% charge and the MacBook turned off again soon after
that.

I decided to go to an apple store for assistance and they couldn't reset
the firmware, so they've sent the laptop for repair under warranty.
It is possible that the battery was malfunctioning by itself and that it is
indeed an hardware problem: however, I think it's important to note that
the laptop has been always working flawlessly up to then (except for the
plausible battery miscalibration), with battery power or plugged in, and
the issue occurred only when doing the calibration.
I will give you a follow up once i Will have more info. I must say, though,
that I'm a bit concerned about using this app again.
Thanks for your time

Alessio


Il Ven 5 Mag 2023, 13:30 masterzone3d ***@***.***> ha scritto:
